bridges/AnimeUltimeBridge.php

```diff
@@ -91,6 +97,8 @@ public function collectData(){
 								0,
 								strpos($item_description, '<div id="table">')
 							);
+
+							// Convert relative image src into absolute image src, remove line breaks
```

Comment:
We have an internal function for this I think

Reply:
Yes, in `html.php` (`defaultLinkTo`). In the case of this bridge, I only added a comment and did not touch existing code, which could have been replaced by a call to this function. However, `defaultLinkTo` does not seems ready yet, as it will not work for all websites because some sites are using a mix of:

 - Full links `https://site.com/some/resource.html`
 - Links relative to the root `/some/resource.html`
 - Links relative to the resource being viewed eg `otherpage.html`

Here is an [example](https://www.futura-sciences.com/tech/actualites/tablette-magicscroll-curieuse-tablette-smartphone-inspiree-parchemin-72660/) of a page containing both links relative to the root and absolute links. The function would prepend domain names to links already containing it, thus breaking some valid links.

In order to solve this problem, the function would need both the website URI e.g. `https://site.com` *and* the current resource URI e.g. `/some/resource.html` (or full URI, that would work too) in order to properly rebuild links relative to the resource being viewed.

If you are interested in that, I could try rewriting the function :slightly_smiling_face:

Reply:
`defaultLinkTo` should work if provided the link of the resource being viewed. `php-urljoin`, which is used internally, is supposed to handle full, relative from provided link and relative from root situations.

If it's broken, please add an example to the test cases: https://github.com/fluffy-critter/php-urljoin/blob/master/tests/tests.php

Reply:
OK, I'm wrong, did not notice that `php-urljoin` actually does that. Fixing the bridge and adding documentation above `defaultLinkTo` to explain what it does :+1:

Reply:
Good point, thanks for taking care of that :+1: 

Although some functions are documented, the core is missing a lot of documentation. Maybe we can get this up to standard (i.e. for phpDocumentor), but that is a separate task for the future. Most functions are explained in the [Wiki](https://github.com/RSS-Bridge/rss-bridge/wiki) at least.